merci/evaluate.py

```diff
@@ -79,12 +79,44 @@ class TransductiveEvaluator(ModelEvaluator):
         [merci.types.Dataset, merci.types.Dataset], merci.types.Dataset
     ]
 
+    @staticmethod
+    def numpy_merger(
+        train: merci.types.Dataset, test: merci.types.Dataset
+    ) -> merci.types.Dataset:
+        """
+        Merge the train and test datasets by concatenating them.
+
+        :param train: A tuple containing input and target data for training
+        :param test: A tuple containing input and target data for testing
+        :return: A tuple containing the merged input and target data
+        """
+        return (
+            np.concatenate([train[0], test[0]]),
+            np.concatenate([train[1], test[1]]),
+        )
+
+    @staticmethod
+    def list_merger(
```

Comment:
Do we still need `list_merger`?

Reply:
I left it in case we want to use some classifiers operating on list objects  (or for testing or anything like that)

Reply:
Okok